notebooks/02.02-The-Basics-Of-NumPy-Arrays.ipynb

```diff
@@ -63,17 +63,15 @@
   {
```

Comment:
<p>output is 7!</p>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>reply 1</p>

Reply:
<p>reply 2</p>

Reply:
<p>reply 3</p>

Reply:
<p>r4</p>

Reply:
<p>r5</p>

Reply:
<p>r6</p>